mentorship ios/Views/Members/SendRequest.swift

```diff
@@ -74,7 +76,7 @@ struct SendRequest: View {
                     Section {
                         if membersViewModel.inActivity {
                             ActivityIndicator(isAnimating: $membersViewModel.inActivity, style: .medium)
-                        } else if !membersViewModel.sendRequestResponseData.success {
+                        } else if !membersViewModel.sendRequestResponseData.success && !isShowing {
```

Comment:
We can retain this change, and replace the isShowing variable with the success variable form view model.